airbyte-integrations/connectors/destination-oracle-strict-encrypt/gradle.properties

```diff
@@ -0,0 +1 @@
+junitMethodExecutionTimeout = 30 m
```

Comment:
can we remove this and only override problematic test methods?

Reply:
will do